### DISCUSSION TITLE
How the Arc3D v3d format defined?

### DISCUSSION THREAD

**Discussion by MineClever** [cnr-isti-vclab/meshlab#909] [Q&A]
the arc3d /V3D format was shown by this video   https://www.youtube.com/watch?v=56wuxAKBhME
which can process in meshlab

but I cant get any information for this format. 

at the same time , we can get depthmap ,camerafile , rgb map and pointCloud by a iphone (with faceid / lidar)
 with the application : https://apps.apple.com/us/app/record3d-3d-videos/id1477716895

ask for helping about this format(arc3d / v3d), thanks!

**Comment by alemuntoni** [comment]
You can get some infos about the arc3d format from these MeshLab papers: 

https://diglib.eg.org/bitstream/handle/10.2312/LocalChapterEvents.ItalChap.ItalianChapConf2008.129-136/129-136.pdf?sequence=1 

https://www.culturalinformatics.org.uk/sites/culturalinformatics.org.uk/files/Rome_Pro_20111012.pdf#page=61
